### DISCUSSION TITLE
Does CLANG work with CPU SIMD?

### DISCUSSION THREAD

**Discussion by chosen-ox** [tinygrad/tinygrad#7574] [Q&A] [answered]
I wonder when I set "CLANG=1" to use CLANG as my backend. Can the code atomatically leverage SIMD in intel CPU? If not, where I can support this? Thank you!

**Comment by nimlgen** [comment, marked as answer]
No SSE/AVX intrinsics generated for now. Depends on llvm's auto-vectorization (which is not really good in our cases). Try with DEBUG>=6 to see asm and maybe different compilers (easy to switch in ops_clang.py)
